lib/beaker/host/unix/pkg.rb

```diff
@@ -120,6 +120,34 @@ def install_package(name, cmdline_args = '', version = nil, opts = {})
     end
   end
 
+  # Extract RPM command's proxy options from URL
+  #
+  # @param [Url] url A URL of form http://host:port
+  #
+  def extract_rpm_proxy_options(url)
+    begin
+      host, port = url.match(/https?:\/\/(.*):(\d*)/)[1,2]
+      raise if host.empty? or port.empty?
+      "--httpproxy #{host} --httpport #{port}"
+    rescue
+      raise "Cannot extract host and port from #{url}"
+    end
+  end
+
+  # Install a package using RPM
+  #
+  # @param [String] name          The name of the package to install.  It may be a
+  #                               filename or a URL.
+  # @param [String] cmdline_args  Additional command line arguments for
+  #                               the package manager.
+  def install_package_with_rpm(name, cmdline_args = '', opts = {})
```

Comment:
please include a doc section for the opts parameters that are taken by this method.  You can see an example of this [here](https://github.com/puppetlabs/beaker/blob/master/lib/beaker/dsl/install_utils/foss_utils.rb#L243-L247).

Also, please include a `@return` doc for both methods.